packages/material-ui/scripts/rollup.config.js

```diff
@@ -31,9 +29,9 @@ export default [
     external: Object.keys(globals),
     plugins: [
       nodeResolve(),
-      babel(getBabelOptions()),
-      commonjs(getCommonjsOptions()),
-      sizeSnapshot(),
```

Comment:
The size output is wrong, it doesn't match what the browser network panel is telling me. But more importantly, it made me realize that development bundle size isn't a metric we want to track. I think that it's just noise.

Reply:
The diff of this pull-request
```diff
-  "build/umd/material-ui.development.js": {
-    "bundled": 994052,
-    "minified": 363508,
-    "gzipped": 94961
-  },
```
vs https://unpkg.com/@material-ui/core@1.2.1/umd/material-ui.development.js

![capture d ecran 2018-06-16 a 16 54 44](https://user-images.githubusercontent.com/3165635/41499741-02033188-7186-11e8-8f7e-401e87e50173.png)

It's wrong in the sense that no minifier is applied to the output, it looks like size-snapshot assume it's going to be applied. But againt, I think that dev bundle size is noise, better only tracking the production bundle :)